purchase_request_department/views/purchase_request_department_view.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This line is no longer needed since v11. Just cosmetic, not a blocker.

Reply:
I read about it, and I found it for .py "For Python 3 (Odoo >= 11.0), no need for utf-8 coding line as this is implicit."
Although certainly this line here could be removed, because the XML declaration is only required if you are using a non-Unicode character encoding (technically, anything other than UTF-8 or UTF-16)